api/app/signals/apps/api/v1/views/signal.py

```diff
@@ -7,7 +7,7 @@
 
 from signals.apps.api import mixins
 from signals.apps.api.generics.filters import FieldMappingOrderingFilter
-from signals.apps.api.generics.permissions import SIAPermissions
+from signals.apps.api.generics.permissions.base import SignalCreateInitialPermission
```

Comment:
probably want this import to be: 
`from signals.apps.api.generics.permissions import SignalCreateInitialPermission`
(see also comment on  signals.apps.api.generics.permissions.base module).